metalearners/metalearner.py

```diff
@@ -133,6 +137,41 @@ def _validate_n_folds_synchronize(n_folds: dict[str, int]) -> None:
         raise ValueError("Need at least two folds to use synchronization.")
 
 
+def _evaluate_model_kind(
+    cfes: Sequence[CrossFitEstimator],
+    Xs: Sequence[Matrix],
+    ys: Sequence[Vector],
+    scorers: Sequence[str | Callable],
+    model_kind: str,
+    is_oos: bool,
+    is_treatment: bool,
```

Comment:
```suggestion
    is_treatment_model: bool,
```
I first thought this had to do with `w=1` or `w=0`. :)